### PR TITLE
Cindy feature

### PR DESCRIPTION
**Pull request by cindyfeldman**
added description of game in pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# Deforestation Game
+# 	*Welcome to the Deforestation Game*
```
Comment (buttercakeee) [inline README.md:1]: italics. fancy

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
...
+##Discover the importance of regulating the manufacturing of paper through a fun choose your story game!
```
Comment (buttercakeee) [inline README.md]: nice description

Comment (Ryguy-1) [reply README.md]: agreed

File: README.md (hunk 1 above)
Comment (kronodeus) [reply README.md:1]: Fancy indeed!

**Pull request merged**